Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -215,10 +243,12 @@ func projectGeneratorTests() {
 
             $0.it("generates dependencies") {
                 let pbxProject = try getPbxProj(spec)
+                
                 let nativeTargets = pbxProject.objects.nativeTargets.referenceValues
                 let dependencies = pbxProject.objects.targetDependencies.referenceValues
-                try expect(dependencies.count) == 1
-                try expect(dependencies.first!.target) == nativeTargets.first { $0.name == framework.name }!.reference
+                try expect(dependencies.count) == 2
+                try expect(dependencies[0].target) == nativeTargets.first { $0.name == framework.name }!.reference
```

Comment:
relying on the order in which target target-dependencies are generated  might not be very resilient.
I did not know how to fix this. 
Actually "resolving" the dependencies and then asserting that they reference the correct  target might be a better aproach.